apps/frontend-manage/src/pages/_document.tsx

```diff
@@ -0,0 +1,253 @@
+// @ts-nocheck
+
+import Document, { Head, Html, Main, NextScript } from 'next/document'
+
+class MyDocument extends Document {
```

Comment:
refactor to functional component instead of class (even if we lose some typescript stuff)

https://nextjs.org/docs/pages/building-your-application/routing/custom-document